src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useContext } from "react";
+import {
+	ACTIONS,
+	EpisodesContext as Episodes,
+} from "COMPONENTS/Episodes/";
+import { TrailerContext as Trailer } from "COMPONENTS/Episode/";
+import UTILS from "SHARED/utils.js";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	// ----------------------------
+	const {
+		state: {
+			activeIndex,
+			overlayActive,
+			episodeCount
+		},
+		dispatch
+	} = useContext(Episodes);
+
+
+	const trailers = useContext(Trailer);
+
+	const { src: prevTrailerSrc } = trailers.prev || {};
+	const { src: nextTrailerSrc } = trailers.next || {};
+
+	// RENDER
+	// ----------------------------
+	const { 
+		episodeId,
+		index,
+		number: episodeNo,
+		title,
+		thumbnail: {
+			src: thumbSrc,
+			description: thumbDesc
+		},
+		trailer: {
+			src: trailerSrc
+		}
+	} = props;
+
+	// handle visibility of active episode and episode preview overlay
+	const isActiveEpisode 	= activeIndex === index;
+	const isOverlayActive   = isActiveEpisode && overlayActive;
+	const hidden 			= !isOverlayActive;
+
+	const trailerSrcAttr    = isOverlayActive ? {source: trailerSrc} : {};
```

Comment:
I get what you're trying to do here, but instead of having `<video src={src}>` we should have
```
<video>
    <source src={src} type="video/mp4" />
</video>
```

and because we're using a `<source>` element, we can just do the conditional rendering down in the return.